plugins/Actions/Reports/Base.php

```diff
@@ -91,7 +92,11 @@ protected function addPageDisplayProperties(ViewDataTable $view)
     protected function addExcludeLowPopDisplayProperties(ViewDataTable $view)
     {
         if (Common::getRequestVar('enable_filter_excludelowpop', '0', 'string') != '0') {
-            $view->requestConfig->filter_excludelowpop = 'nb_hits';
+            if (Common::getRequestVar('flat', 0, 'int') === 1) {
+                $view->requestConfig->filter_excludelowpop = 'nb_hits';
+            } else {
+                $view->requestConfig->filter_excludelowpop = Metrics::INDEX_PAGE_NB_HITS;
```

Comment:
un-related to this PR, but +1 toconsider removing this mix of Metrics ID and Named metrics... could you create an issue for this? (to store named metrics in archives rather than stored metrics ID)